spec/system/distribution_system_spec.rb

```diff
@@ -409,7 +501,7 @@
       end
 
       expect(page).to have_content("Sorry, we weren't able to save")
-      find_all(".numeric")[0].set 1
+      find_all(".numeric")[2].set 1
```

Comment:
Is there a reason we're addressing elements by number instead of getting the actual class, ID etc of it?